template3a-tarot.js

```diff
@@ -1,27 +1,39 @@
 // This IIFE should create a subclass (constructor TarotCard)
 //  of a superclass (constructor Card)
-var TarotCard = (function(){
-
-	// Create subclass constructor:
-	function Ctor(id) {
-		// call superclass ctor first,
-		// then add subclass-specific properties
-	}
-	// Replace default prototype so that subclass inherits from superclass...
-
-
-
-
-	// Override any inappropriate inherited instance methods...
-
-
-
-
-	// Instance methods are inherited, but class methods and private arrays aren't;
-	//  you'll need to create corresponding methods/arrays for this subclass...
-
-
-
-
-	return Ctor; //== TarotCard
+var TarotCard = (function() {
```

Comment:
Your TarotCard function is looking good. Try improving it by adding functionality where, if a card is not valid, it will not give the user any pertinent information, like rank or suit or name.